sonar-plugin-api/src/test/java/org/sonar/api/security/ExternalGroupsProviderTest.java

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Please ignore the changes in these files. These changes are done as part of the SONAR-6856 ( We have created separate pull request for this)

SONAR-6856: Add a new API ExternalGroupsProvider.doGetGroups(Context context) to allow plugins to pass groups information to SonarQube